src/customize-pop/default-applications.md

```diff
@@ -0,0 +1,83 @@
+# Default Applications
+
+Options to set the default web browser, email client, calendar application, music player, video player, and photo viewer are located in `Settings` ➞ `Default Applications`. This menu allows setting a single application as the default handler for a larger number of similar file types.
+
+![Default Applications](/images/application-settings/default-applications.png)
+
+Designate an application as the default by selecting it from the dropdown menu. If no appropriate application is installed to handle a media type, the dropdown selection will be grayed out. <!--link to installing applications section when complete-->
+
+![Select Default](/images/application-settings/select-default.png)
```

Comment:
This screenshot's red highlight box makes it kind of difficult to see why Firefox is listed (especially since a user probably wouldn't think of it as a photo viewer.) I would suggest extending the box to include the entire drop-down; it's clear which one's currently highlighted.

Reply:
Fixed